code/__HELPERS/logging/atmos.dm

```diff
@@ -1,8 +1,5 @@
 /// Logs the contents of the gasmix to the game log, prefixed by text
```

Comment:
make this use the data list

Reply:
What is the data list used for?

Reply:
The data list is your user defined map of key(string) to value(can be anything  but is converted to string anyway) which is written in json format to your log file.

The name of your file where this data is stored is <our usual log file>-json.json.  obviously json format can be parsed later for statictics and what not

In your case vars like temperature and moles can be put in list format as 2 seperate keys and passed to log game.

Message should only be used to explain what vars are being logged and not store the actual data itself

Reply:
Done.